<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>opcionProductos</class>
 <widget class="QDialog" name="opcionProductos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1343</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Opciones:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbtAgInventario">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Agregar inventario</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAgNuevoProd">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Agregar nuevo producto</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnModificarProd">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Modificar producto</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtBorrarFac">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Borrar facturas</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtNumFac">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Establecer n√∫mero de facturas</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtAgregarCliente">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Agregar cliente</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Salir</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
